src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.scss

```diff
@@ -1,5 +1,4 @@
-@import '../../../shared/datatable/table/table.component.scss';
 
-td.wrap {
```

Comment:
I would suggest keeping this, otherwise some cells will have its content cut.

Reply:
Okay, I'm going to re-add it, thanks.